CRM/Financial/BAO/FinancialAccount.php

```diff
@@ -246,4 +246,43 @@ public static function getFinancialAccountForFinancialTypeByRelationship($financ
     return Civi::$statics[__CLASS__]['entity_financial_account'][$financialTypeID][$relationTypeId];
   }
 
+  /**
+   * Get Financial Account type relations
+   *
+   * @param $flip bool
```

Comment:
Isn't that the same as "Exchanges all keys with their associated values in an array"

http://php.net/manual/en/function.array-flip.php